content/applications/finance/fiscal_localizations/ecuador.rst

```diff
@@ -61,6 +61,21 @@ localization:
        <../accounting/customer_invoices/electronic_invoicing>`, based on the Technical
        documentation published by the SRI. The authorized documents are: Invoices, Credit Notes,
        Debit Notes, Withholdings and Purchase liquidations.
+   * - :guilabel:`Ecuadorian Accounting Reports`
+     - `l10n_ec_reports`
+     - Includes all the technical and functional requirements to generate forms 103 and 104.
+   * - :guilabel:`Ecuadorian ATS Report`
```

Comment:
```suggestion
   * - :guilabel:`Ecuador - ATS Report`
```

Reply:
@auva-odoo @samueljlieber I agree with this suggestion :)